staker-rewards/package.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
As we initiate a new NPM package in `staker-rewards` directory can we also configure JS linting? Defining this now will be super handy for following PRs.